controller/appcontroller.go

```diff
@@ -1777,6 +1778,18 @@ func (ctrl *ApplicationController) canProcessApp(obj interface{}) bool {
 	if !ok {
 		return false
 	}
+	if annotations := app.GetAnnotations(); annotations != nil {
+		if skipVal, ok := annotations[common.AnnotationKeyAppSkipReconcile]; ok {
+			if skipReconcile, err := strconv.ParseBool(skipVal); err == nil {
+				if skipReconcile {
+					log.Infof("Skipping Application reconcile based on annotation %s", common.AnnotationKeyAppSkipReconcile)
```

Comment:
If we really want to log here, I think we should set up some context for this log entry - e.g. the application name.

You can use `log.WithField("application", app.QualifiedName()).Infof(...)` for that. 

However - this function could be called quite often, and that might result in some log flood. I would change log level to debug for this message. And also, either remove the below warning log statement as well or transform it to debug.

Reply:
Done as suggested.